### PR TITLE
Add python tests for multi-input CropMirrorNormalize

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

File: dali/test/python/test_pipeline.py [hunk 1]
```diff
@@ -279,6 +279,39 @@ def iter_setup(self):
...
+                                                crop = (224, 224),
+                                                image_type = types.RGB,
+                                                mean = [128., 128., 128.],
+                                                std = [1., 1., 1.])
+
+        def define_graph(self):
+            inputs, labels = self.input(name="Reader")
+            images = self.decode(inputs)
+            images2 = self.decode2(inputs)
+            output1, output2 = self.cmnp([images.gpu(), images2.gpu()])
```
Comment (mzient) [inline dali/test/python/test_pipeline.py]:
```suggestion
            output1, output2 = self.cmnp([images.gpu(), images2.gpu()])
            output3 = self.cmnp(images.gpu())
            output4 = self.cmnp(images2.gpu())
```

File: dali/test/python/test_pipeline.py [hunk 2]
```diff
@@ -279,6 +279,39 @@ def iter_setup(self):
...
+            return (output1, output2)
+
+        def iter_setup(self):
+            pass
+    pipe = HybridPipe(batch_size=batch_size)
+    pipe.build()
+    for _ in range(10):
+        out1, out2 = pipe.run()
+        check_batch(out1.as_cpu(), out2.as_cpu(), batch_size)
```
Comment (mzient) [inline dali/test/python/test_pipeline.py]:
```suggestion
        check_batch(out1.as_cpu(), out3.as_cpu(), batch_size)
        check_batch(out2.as_cpu(), out4.as_cpu(), batch_size)
```

File: dali/test/python/test_pipeline.py [hunk 3]
```diff
@@ -279,6 +279,39 @@ def iter_setup(self):
...
+            images2 = self.decode2(inputs)
+            output1, output2 = self.cmnp([images.gpu(), images2.gpu()])
+            return (output1, output2)
+
+        def iter_setup(self):
+            pass
+    pipe = HybridPipe(batch_size=batch_size)
+    pipe.build()
+    for _ in range(10):
+        out1, out2 = pipe.run()
```
Comment (mzient) [inline dali/test/python/test_pipeline.py]:
```suggestion
        out1, out2, out3, out4 = pipe.run()
```

**Comment by jantonguirao** [comment]
!build

File: dali/test/python/test_pipeline.py [hunk 4]
```diff
@@ -279,6 +279,51 @@ def iter_setup(self):
...
+                                                image_type = types.RGB,
+                                                mean = [128., 128., 128.],
+                                                std = [1., 1., 1.])
+
+        def define_graph(self):
+            inputs, labels = self.input(name="Reader")
+            images = self.decode(inputs)
+            images2 = self.decode2(inputs)
+
+            images_device  = images if self.device == "cpu" else images.gpu()
```
Comment (JanuszL) [inline dali/test/python/test_pipeline.py]: This is not needed I think. As CPU op will accept `images.gpu()` as well.

File: dali/test/python/test_pipeline.py [hunk 5]
```diff
@@ -279,6 +279,51 @@ def iter_setup(self):
...
+
+            output1, output2 = self.cmnp([images_device, images2_device])
+            output3 = self.cmnp([images_device])
+            output4 = self.cmnp([images2_device])
+            return (output1, output2, output3, output4)
+
+        def iter_setup(self):
+            pass
+
+    for device in ["cpu", "gpu"]:
```
Comment (JanuszL) [inline dali/test/python/test_pipeline.py]: CPU and GPU data is not compared one to another?

Comment (jantonguirao) [reply dali/test/python/test_pipeline.py]: not in this test, but I believe that this is covered in the C++ tests

File: dali/test/python/test_pipeline.py (hunk 4 above)
Comment (jantonguirao) [reply dali/test/python/test_pipeline.py]: Even if not needed, I think it is weird if we write `.gpu()` on the input of a CPU operator. What do you think?

Comment (JanuszL) [reply dali/test/python/test_pipeline.py]: ok

**Pull request merged**